subsys/kafka/src/main/conf/conf.d/kafka.conf

```diff
@@ -1,14 +1,7 @@
 [kafka]
-# By default, this is disabled
+# By default, this is enabled
 #
-#enabled=false
-
-bootstrap.servers=
-security.protocol=SSL
-ssl.keystore.location=kafka.client.jks
-ssl.keystore.password=pass
-ssl.key.password=pass
-ssl.truststore.location=truststore.jks
-ssl.truststore.password=pass
-key.serializer=org.apache.kafka.common.serialization.StringSerializer
-value.serializer=org.apache.kafka.common.serialization.StringSerializer
\ No newline at end of file
+enabled=true
+kafka.bootstrap.servers=127.0.0.1:9092
+kafka.topics=store-event
+kafka.group=kstreams-group
```

Comment:
We should also consider the security like the log-kafka did. Like authentication and ssl support.

Reply:
@ligangty I didn't get a chance to connect the Kafka operator instance for any testing yet, I think we can consider configuring this after it's verified during integration testing.